.circleci/config.yml

```diff
@@ -18,6 +18,14 @@ parameters:
   release_build:
     type: boolean
     default: false
+  # Display name of the app and should match the usual public facing name (e.g. WordPress, Jetpack)
+  app_name:
+    type: string
+    default: "WordPress"
+  # The key used to reference the app in the Fastfile
+  product_key:
+    type: string
+    default: "wordpress"
```

Comment:
I don't think these are modified at all, are they? I see there is a Jetpack build, but I don't think it's actually using these information at all.

Also, I am concerned about using a default for these. It saves a few keystrokes when we have all the context, but when we don't, it'd help if CircleCI complains about the value missing so we deliberately pick the right one.

Reply:
They're being used when setting the parameters in the Release Build but we could just use the pipeline parameters directly in the job if I'm not mistaken so I'll clean that up and I agree with the idea behind removing the default value here 👍